app/graphql/mutations/create_submission_comment.rb

```diff
@@ -13,6 +13,17 @@ def resolve(_params)
           comment: @params[:comment],
           user_id: current_user.id
         )
+
+      StartupMailer
+        .comment_on_submission(submission, comment, current_user)
+        .deliver_later if submission.students.pluck(:id).exclude?(current_user.id)
```

Comment:
Invalid comparison of student ID against user ID. Currently, this causes an email to be sent to self when commenting on one's own submission.